tapgarden/planter.go

```diff
@@ -571,6 +571,32 @@ func (c *ChainPlanter) fundGenesisPsbt(ctx context.Context,
 			batchKey[:], feeRate.FeePerKVByte().String())
 	}
 
+	minRelayFee, err := c.cfg.Wallet.MinRelayFee(ctx)
+	if err != nil {
+		return nil, fmt.Errorf("unable to obtain minrelayfee: %w", err)
+	}
+
+	// If the fee rate is below the minimum relay fee, we'll
+	// bump it up.
+	if feeRate < minRelayFee {
+		switch {
+		// If a fee rate was manually assigned for this batch, we err
+		// out, otherwise we silently bump the feerate.
+		case manualFeeRate != nil:
+			// This case should already be handled by the
+			// `checkFeeRateSanity` of `rpcserver.go`. We check here
+			// again to be safe.
+			return nil, fmt.Errorf("feerate does not meet " +
+				"minrelayfee")
+		default:
+			log.Infof("bump fee rate for batch %x to meet "+
```

Comment:
nit: Log statements usually start capitalized (vs. error messages that should be lower case according to Golang defaults and linter preferences).